internal/controller/nginx/config/http/config.go

```diff
@@ -119,11 +119,12 @@ const (
 
 // Upstream holds all configuration for an HTTP upstream.
 type Upstream struct {
-	Name      string
-	ZoneSize  string // format: 512k, 1m
-	StateFile string
-	KeepAlive UpstreamKeepAlive
-	Servers   []UpstreamServer
+	Name                string
+	ZoneSize            string // format: 512k, 1m
+	StateFile           string
+	KeepAlive           UpstreamKeepAlive
+	LoadBalancingMethod string
```

Comment:
Curious why the fieldalignment linter isn't complaining about this struct. I would think this field would need to be nested next to the other strings.